### DISCUSSION TITLE
Allure Report Azure DevOps extension stuck on loading the report when there are larger amounts of failed tests

### DISCUSSION THREAD

**Discussion by ciconq** [allure-framework/allure2#2723] [Bug Reports]
Hello, 

I am monitoring an issue related to `Allure Report Azure DevOps extension` when there are larger amounts of failed tests. 
For example I am having `300` tests and using `VSTest@2` task with `rerunMaxAttempts` set to `5` for `failed` tests.
Imagine all of your tests fail due to driver incompatibility or environment issues. This will result in `1800` test failures. 

After the test run is done and I try to open the `Allure Report` I am stuck on the following state which doesn't change at all: 
![Screenshot 2024-09-09 171604](https://github.com/user-attachments/assets/5a3a1742-72f8-4967-8eb3-5ac9f10ff4ae)

All those `110` chunks are fetched for like `5-7` minutes which is an additional issue but the one described above is a bigger one as the report never loads and is just stuck on the `Loading the report(110/110 chunks)` part which results in unavailability to view the report.

Also the `network` tab doesn't show anything wrong.

**Comment by epszaw** [comment]
Hey there! Unfortunately, it's not possible to render a report 512mb or more in the browser using the current implementation. [There is a notice about it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length#description).

Stay in touch, we're trying to resolve the problem.
